JSONResourceFile.js

```diff
@@ -36,13 +37,18 @@ var logger = log4js.getLogger("loctool.plugin.JSONResourceFile");
  * @param {Object} props properties that control the construction of this file.
  */
 var JSONResourceFile = function(props) {
+    var lanDefaultLocale, propsLocale;
     this.locale = new Locale();
-    this.fallbackLocale = false;
+    this.baseLocale = false;
 
     if (props) {
         this.project = props.project;
         this.locale = new Locale(props.locale);
         this.API = props.project.getAPI();
+
+        langDefaultLocale = new LocaleMatcher({locale: this.locale.language});
+        propsLocale = new LocaleMatcher({locale: this.locale});
+        this.baseLocale = ((langDefaultLocale.getLikelyLocale().getSpec() === propsLocale.getLikelyLocale().getSpec())? true:false);
```

Comment:
Don't need the question-colon operator here:

```javascript
this.baseLocale = langDefaultLocale.getLikelyLocale().getSpec() === propsLocale.getLikelyLocale().getSpec();
```

Reply:
I updated it